### PR TITLE
Vacancy-Placement creation and how to associate candidates

### PR DESCRIPTION
**Pull request by robeecaneda1**

### CODE REVIEW

File: VacancyCreation.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- Contact: The person that initially sent us the Vacancy (someone from Company or Partner)
+- Company: Usually a Company that has Company Type "End Customer". The Candidate will perform services for this Company.
+- Partner: The Company we received the Vacancy from (usually a Company Type "Mid-market")
+- Consultant: The itenium employee that sent the Vacancy
+- Seniority:  
+	- Junior: 0-2 years
+	- Medior: 2-5 years
+	- Senior: 5+ years
+	- Architect: Not really based on years of XP
+- Job Category: See `Candidate-Categories.md` for more information for this field
```
Comment (Laoujin) [inline VacancyCreation.md]: Does this tutorial still exist for Zoho?

Comment (robeecaneda1) [reply VacancyCreation.md]: Yes, it does. It's still part of the steps when we create a job opening.

File: VacancyCreation.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+﻿Job Opening
+=======
+
+Always pick "Contract".
```
Comment (Laoujin) [inline VacancyCreation.md:4]: Does this field still exist in Zoho? I cannot find it?

File: VacancyCreation.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- Contact: The person that initially sent us the Vacancy (someone from Company or Partner)
+- Company: Usually a Company that has Company Type "End Customer". The Candidate will perform services for this Company.
```
Comment (Laoujin) [inline VacancyCreation.md:7]: Company does not exist in Zoho, it is called "Client Name"

File: VacancyCreation.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- Partner: The Company we received the Vacancy from (usually a Company Type "Mid-market")
+- Consultant: The itenium employee that sent the Vacancy
```
Comment (Laoujin) [inline VacancyCreation.md:9]: Consultant does not exist on this page.

File: VacancyCreation.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- Seniority:  
+	- Junior: 0-2 years
+	- Medior: 2-5 years
+	- Senior: 5+ years
+	- Architect: Not really based on years of XP
```
Comment (Laoujin) [inline VacancyCreation.md:14]: Seniority does not exist on this page

File: VacancyCreation.md (hunk 2 above)
Comment (robeecaneda1) [reply VacancyCreation.md:4]: Yes, it's under Job Type. I'll update the guide to make it specific

File: VacancyCreation.md (hunk 5 above)
Comment (robeecaneda1) [reply VacancyCreation.md:14]: It's under "Work Experience"

File: VacancyCreation.md (hunk 3 above)
Comment (robeecaneda1) [reply VacancyCreation.md]: updated Company to Client

File: VacancyCreation.md (hunk 4 above)
Comment (robeecaneda1) [reply VacancyCreation.md]: changed consultant to Assigned Recruiter

**Pull request merged**